iModelJsNodeAddon/presentation/ECPresentationSerializer.h

```diff
@@ -128,4 +131,18 @@ struct DefaultClassSerializer : IECClassSerializer
 	virtual rapidjson::Document _SerializeECClass(ECClassCR, rapidjson::Document::AllocatorType&) override;
 };
 
+/*=================================================================================**//**
+* @bsiclass
++===============+===============+===============+===============+===============+======*/
+struct IModelJsBoundQueryValueSerializer : IBoundQueryValueSerializer
+{
+    ~IModelJsBoundQueryValueSerializer() {}
+    rapidjson::Document _ToJson(BoundQueryECValue const&, rapidjson::Document::AllocatorType*) const;
+    rapidjson::Document _ToJson(BoundQueryId const&, rapidjson::Document::AllocatorType*) const;
+    rapidjson::Document _ToJson(BoundQueryIdSet const&, rapidjson::Document::AllocatorType*) const;
+    rapidjson::Document _ToJson(BoundECValueSet const&, rapidjson::Document::AllocatorType*) const;
+    rapidjson::Document _ToJson(BoundRapidJsonValueSet const&, rapidjson::Document::AllocatorType*) const;
+    std::unique_ptr<BoundQueryValue> _FromJson(BeJsConst const&);

```

Comment:
missing `override` specifiers. did you compile this code?

Reply:
Looks like for the compiler these specifiers are optional? Added them both here and in DefaultBoundQueryValueSerializer.

Reply:
I don't remember, they might be optional for Windows compiler. Definitely not optional for clang.